lib/assert.js

```diff
@@ -62,10 +64,9 @@ exports.AssertionError = AssertionError;
 function getStack() {
 	const limitBefore = Error.stackTraceLimit;
 	Error.stackTraceLimit = Infinity;
-	const obj = {};
-	Error.captureStackTrace(obj, getStack);
+	const error = new Error();
```

Comment:
I'm not sure how this impacts the captured stack trace. Perhaps the `AssertionError` constructor needs to do this `stackTraceLimit` dance, and explicitly capture the stack trace excluding the `getStack()` function?

Reply:
That would mean getting rid of `getStack()` no?